pyproject.toml

```diff
@@ -1,47 +1,89 @@
+[project]
+name = "dune-sync"
+dynamic = ['version', 'dependencies']
+
+[tool.poetry]
+name = "dune-sync"
+version = "0.2.0"
```

Comment:
I don't really want to have to update this manually. Can it be set to 0.0.0 or left out?

Reply:
My bad, this is superfluous. The version is declared as a `dynamic` property and that's enough to make the tooling infer it from git:
```toml
[project]
name = "dune-sync"
dynamic = ['version', 'dependencies']
```